src/utils/schema/resolvers/Query/all.ts

```diff
@@ -259,6 +259,7 @@ export default (entityName: string, fieldsByName: any, idsByName: any) =>
     let linkedRecordsAggregation = [];
     for (const resource of resourcesToQuery) {
       // Build linked records aggregations
+      // AB#35599
```

Comment:
@ketanreliefapps the problem there is that the pipeline is not working on all environments. indeed, $lookup aggregation with let or pipeline fields does not work in CosmosDB ( the dbs that we have for WHO )
so we need to find a solution that would do the same thing, so a similar aggregation, that would work on CosmosDB

this doc indicates what is supported or not:
https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/feature-support-42